.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+## Overview
+**TL;DR**
+<One to two sentence description of the issue you are encountering or trying to solve.>
+
+### Extra Detail
+
+#### Screenshots
+<If bug fix, please upload images of the error to better showcase the problem.>
+
+#### Reasoning
+<If new idea or direction, please create a checklist of proposed ideas or next steps. [Example](https://github.com/github/training-kit/issues/415).>
+
+### How to Contribute
+- [ ] Comment on this issue if you'd like to work/collaborate on it! One of the training-kit maintainers will then "Assign" you to the issue.
+- [ ] Fork the [repository](https://github.com/github/training-kit).
+- [ ] Use the [GitHub Flow](https://guides.github.com/introduction/flow/) to make changes to your fork. 
+- In your command line:
```

Comment:
@hollenberry Should we explain that these steps are so they can test their changes locally?